### ISSUE TITLE
[Installation Error] go-sqlite3 requires cgo

### ISSUE THREAD

**Issue by MushiTheMoshi** [c9s/bbgo#1010]
Hi there,

Thank you so much for all the work done here.

I am still stuck installing the tool, downloaded the binary from releases (bbgo-v1.42.0-linux-amd64.tar.gz), bypassed the otp as suggested in a different ticket, now I am getting the below error on the setup wizard.

```.bash
Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

![image](https://user-images.githubusercontent.com/13651419/201394643-9bffc747-963f-449c-8922-388a3bdc3b13.png)

Is it mandatory MYSQL?

Any help will be very much appreciate it.

Regards,
Julio

**Comment by c9s** [comment]
yeah, it looks like the binary is not compiled with CGO_ENABLED=0, are you using the binary from the release ?

**Comment by MushiTheMoshi** [comment]
yes, I am using the bash script in the readme

**Comment by c9s** [comment]
Ok i think for now you need to use mysql instead of sqlite

I will fix the sqlite build issue in the next few days.

Julio Guerrero ***@***.***>於 2022年11月24日 週四，下午6:51寫道：

**Comment by MushiTheMoshi** [comment]
I did use mysql though there is a different error now when setting up exchange keys:

![image](https://user-images.githubusercontent.com/13651419/203873133-7efd9969-abe4-4ec6-83f6-3572c614e06b.png)

![image](https://user-images.githubusercontent.com/13651419/203873144-8d78287c-2c96-4ba2-9d67-c0bc120918e6.png)

**Comment by c9s** [comment]
did the console log print something?

**Comment by lanphan** [comment]
@c9s 

In order to get rid of CGO, we must replace go-sqlite3 by [modernc.org/sqlite](https://gitlab.com/cznic/sqlite), the performance of new library is twice as slow comparing to go-sqlite3 (benchmark performance report is rather old in 2022, here it is: https://datastation.multiprocess.io/blog/2022-05-12-sqlite-in-go-with-and-without-cgo.html ). What do you think?
